secrets/openldap/Dockerfile.centos7

```diff
@@ -0,0 +1,26 @@
+FROM jdeathe/centos-ssh:2.6.1
```

Comment:
I wonder if we should just publish the docker image on dockerhub instead of having folks build it?

Reply:
Sounds like a great idea especially towards future promotion and reuse!

Reply:
I am not sure who has the ability to publish such a container image, but I don't think that I do.

Is it okay to use this as-is for now with a goal to eventually build a docker image going forward?